openehr-rm/src/main/java/com/nedap/archie/rm/composition/Instruction.java

```diff
@@ -91,7 +91,7 @@ public void setWfDefinition(@Nullable DvParsable wfDefinition) {
     public void setActivities(List<Activity> activities) {
         this.activities = activities;
         for (Activity activity : activities) {
-            setThisAsParent(activity, "activity");
+            setThisAsParent(activity, "activities");
```

Comment:
You also need to fix line 100 with the exact same fix for this to work :)

Reply:
Sure :)